app/helpers/admin/enterprises_helper.rb

```diff
@@ -47,7 +47,8 @@ def build_enterprise_side_menu_items(is_shop, show_properties, show_shipping_met
         { name: 'inventory_settings', icon_class: "icon-list-ol", show: is_shop },
         { name: 'tag_rules', icon_class: "icon-random", show: is_shop },
         { name: 'shop_preferences', icon_class: "icon-shopping-cart", show: is_shop },
-        { name: 'users', icon_class: "icon-user", show: true }
+        { name: 'users', icon_class: "icon-user", show: true },
+        { name: 'white_label', icon_class: "icon-leaf", show: true }
```

Comment:
If you keep a comma `,` at the end of the last element then that doesn't need changing when the next element is added. It makes the diff smaller and easier to review (besides adding elements is easier, fewer merge conflicts).

Reply:
I agree. I thought it warn rubucop, but actually no. Modified then.